modules/shapes/module.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Elementor\Modules\Shapes;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Module extends \Elementor\Core\Base\Module {
+
+	/**
+	 * @param bool $add_custom Determine if the output should include the `Custom` option.
+	 *
+	 * @return array List of paths.
+	 */
+	public static function get_paths( $add_custom = true ) {
```

Comment:
Why should it be here and not on the widget itself?

Reply:
I think that everything which is SVG-related should be managed from the Shapes module in order to make it available to other widgets which might need it in the future.